app/models.py

```diff
@@ -2152,13 +2152,19 @@ def get_job_count(self):
             )
         ).count()
 
+    def get_has_jobs(self):
```

Comment:
Could this be called `has_been_used`, to follow your description? If not, should it be `has_jobs` rather than `get_has_jobs`? Doesn't seem like something you'd think you could set.

Reply:
@tombye I had a think about this and decided the best thing to do was turn this method (and the one above it) into a property. See  313b69f9